### PR TITLE
GameDB: Japanese entries Batch 1

### PR DESCRIPTION
**Pull request by RedDevilus**
### Description of Changes

Adds a bunch of gamefixes and upscaling fixes to games as well as just entries.
- 0 Story
- Amagami
- Bleach Blade Battlers 2nd
- Chobits
- D.C. Da Capo The Origin
- Pandora - Kimi no Namae o Boku wa Shiru
- Katekyoo Hitman Reborn! Nerae!? Ring x Vongola Trainers
- Kateikyoushi Hitman Reborn!! Let's Ansatsu! Nerawareta 10 Daime!
- Kino no Tabi - The Beautiful World
- Kyuuketsu-hime Yui - Sen'yashou
- Shikigami no Shiro - Nanayozuki Gensoukyoku
- Secret of Evangelion

Pictures:

Amagami;
Before:
AmagamiBefore
After:
AmagamiAfter

Bleach Blade Battlers 2nd;
Before:
BleachBladeBattlersBefore
After:
BleachBladeBattlersAfter

Chobits;
Before:
ChobitsBefore
After:
ChobitsAfter

DC Da Capo Origin;
Before:
DCDaCapoBefore
After:
DCDaCapoAfter

Kateikyoushi Hitman Reborn!! Let's Ansatsu! Nerawareta 10 Daime!;
Before:
KateikyoushiBefore
After:
KateikyoushiAfter

Katekyoo Hitman;
Before:
KatekyoBefore
After:
KatekyoAfter

Kino no Tabi - The Beautiful World;
Before:
KinoBefore
After:
KinoAfter

### Rationale behind Changes

Less tinkering
### Suggested Testing Steps

Test relevant games

### CODE REVIEW

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -24689,6 +24774,9 @@ SLPM-65257:
 SLPM-65260:
   name: "Air Land Force"
   region: "NTSC-J"
+SLPM-65261:
+  name: "TBS All-Star Kansha Matsuri Vol .1 - Chou Gouka! Quiz Ketteiban"
```
Comment (Tokman5) [inline bin/resources/GameIndex.yaml:24778]: ``Kansha-sai`` is better.

Comment (RedDevilus) [reply bin/resources/GameIndex.yaml:24778]: I used redump listing:
http://redump.org/disc/38164/

Ironically on psxdatabase:
https://psxdatacenter.com/psx2/games2/SLPM-65261.html

On top it's different to official naming:
image

Serialstation:
https://www.serialstation.com/titles/SLPM/65261

But if you look at the title Kansha Masture means Appreciation Festival and Kansha-sai is more like Thanksgiving Festival.

I can change it though.

**Review by Mrlinkwii — APPROVED** [review]

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -5676,6 +5683,10 @@ SCPS-19328:
 SCPS-19329:
   name: "Bleach - Blade Battlers [PlayStation 2 The Best]"
   region: "NTSC-J"
+  gameFixes:
+    - OPHFlagHack # Special moves "Bankai" hang otherwise
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:5687]: Period

File: bin/resources/GameIndex.yaml [hunk 3]
```diff
@@ -5298,6 +5303,8 @@ SCPS-15119:
   compat: 5
   gameFixes:
     - OPHFlagHack # Special moves "Bankai" hang otherwise
```
Comment (refractionpcsx2) [inline bin/resources/GameIndex.yaml:5305]: Can you add a period here?

**Comment by refractionpcsx2** [comment]
Rebase please :)

**Pull request merged**